lib/dev_68/leds.ex

```diff
@@ -0,0 +1,197 @@
+defmodule Dev68.LEDs do
+  use GenServer
+
+  alias Circuits.GPIO
+  alias RGBMatrix.Engine
+
+  defmodule State do
+    @moduledoc false
+    defstruct [:ic1, :ic2, :iicrst, :sdb, :paint_fn]
+  end
+
+  # P9_11, gpio 30
+  @iicrst_pin 30
+
+  # P9_13 lo, gpio 31
+  @sdb_pin 31
+
+  # The LED matrix circuit layout on the dev-68.
+  # Each atom represents 3 LEDs, unless specified by a {:pad, n} tuple.
+  @matrix_a [
+    [:l001, :l017, :l033, :l046, :l059, {:pad, 1}],
+    [:l002, :l018, :l034, :l047, :l060, {:pad, 1}],
+    [:l003, :l019, :l035, :l048, :l061, {:pad, 1}],
+    [:l004, :l020, :l036, :l049, :none, {:pad, 1}],
+    [:l005, :l021, :l037, :l050, :none, {:pad, 1}],
+    [:l006, :l022, :l038, :l051, :l062, {:pad, 1}],
+    [:l007, :l023, :l039, :l052, :none, {:pad, 1}],
+    [:l008, :l024, :l040, :l053, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}]
+  ]
+
+  @matrix_b [
+    [:l009, :l025, :l041, :l054, :none, {:pad, 1}],
+    [:l010, :l026, :l042, :l055, :l063, {:pad, 1}],
+    [:l011, :l027, :l043, :l056, :l064, {:pad, 1}],
+    [:l012, :l028, :l044, :none, :l065, {:pad, 1}],
+    [:l013, :l029, :none, :none, :none, {:pad, 1}],
+    [:l014, :l030, :l045, :l057, :l066, {:pad, 1}],
+    [:l015, :l031, :none, :l058, :l067, {:pad, 1}],
+    [:l016, :l032, :none, :none, :l068, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}],
+    [:none, :none, :none, :none, :none, {:pad, 1}]
+  ]
+
+  @bus_name "i2c-1"
+  @ic1_addr 0x50
+  @ic2_addr 0x53
+
+  # Client
+
+  def start_link([]) do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  def paint(frame) do
+    GenServer.cast(__MODULE__, {:paint, frame})
+  end
+
+  # Server
+
+  @impl GenServer
+  def init([]) do
+    # TODO: remove me: this should be in some "Xebow"-like application
+    layout = Dev68.Layout.layout
+    leds = KeyboardLayout.leds(layout)
+    animation = RGBMatrix.Animation.new(RGBMatrix.Animation.SolidReactive, leds)
+    RGBMatrix.Engine.set_animation(animation)
+    # TODO: end remove me
```

Comment:
This is just temporary until we can make a place for controlling which animation is active.